yaml-tests/src/test/resources/vector.yamsql

```diff
@@ -0,0 +1,55 @@
+#
+# vector.yamsql
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2021-2024 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+options:
+  supported_version: !current_version
+---
+schema_template:
+    create table tHalfVector(a bigint, b vector(3, half), primary key(a))
+    create table tFloatVector(a bigint, b vector(3, float), primary key(a))
+    create table tDoubleVector(a bigint, b vector(3, double), primary key(a))
```

Comment:
I don't see why it wouldn't work, but can you add a test that has multiple vectors in the same table? 
Of the same type, of different types? 
Can you add an array of vectors, and a struct with vectors in it?